src/core/src/opsets/opset.cpp

```diff
@@ -4,11 +4,14 @@
 
 #include "ngraph/opsets/opset.hpp"
 
+#include "../itt.hpp"
```

Comment:
```suggestion
#include "itt.hpp"
```
I guess it should work